### ISSUE TITLE
Getting HTTP 405 when trying to register

### ISSUE THREAD

**Issue by adriaanjoubert** [vascokk/fullstack-rust#3]
Hi @vascokk, not sure if I'm missing something, but I ran the commands in the README and I'm getting a `POST http://127.0.0.1:8088/register/a/a 405 (Method Not Allowed)` when trying to register a user and a color. Are you able to run the project from the master branch?

**Comment by vascokk** [comment]
Yes, I am. Just, use 'X' and 'O' for colors, otherwise, the game board doesn't update the colors after the move (a bug!). Also, there could be a stale active game, so either truncate the tables or update all the existing games to ended=true
